lib/network/api/resource/resource.go

```diff
@@ -30,10 +30,12 @@ func NewResourceList(list []Resource, selfLink, nextLink, prevLink string) *Reso
 
 func (l ResourceList) Resource() *hal.Resource {
 	rl := hal.NewResource(struct{}{}, l.LinkSelf())
+
+	var rCollection hal.ResourceCollection
 	for _, apiResource := range l.Resources {
-		r := apiResource.Resource()
-		rl.Embed("records", r)
+		rCollection = append(rCollection, apiResource.Resource())
 	}
+	rl.EmbedCollection("records", rCollection)
```

Comment:
@kfangw  I changed this.  Because  `rl.Embed` with 1 element doesn't make array `"records":[]`.. :-> 

```
```{"_embedded":{"records":{"_links":{"self":{"href":"/operations/6BTePtJ5SkWD2togQE6SPVuCirArUHkrELBGm75F2CVS-FSFsCjzLB1ZdrDTwkvRSRFKuPywM92xAxdtKAPGbCoHn"},"transactions":{"href":"/transactions/FSFsCjzLB1ZdrDTwkvRSRFKuPywM92xAxdtKAPGbCoHn"}},"account":"","amount":"3081","funder":"GBNRXQCRGN3ANFKAYUD5IX45X7JZ3UVZLIX33IVJN65L7KE4WFL5A53B","hash":"6BTePtJ5SkWD2togQE6SPVuCirArUHkrELBGm75F2CVS-FSFsCjzLB1ZdrDTwkvRSRFKuPywM92xAxdtKAPGbCoHn","id":"6BTePtJ5SkWD2togQE6SPVuCirArUHkrELBGm75F2CVS-FSFsCjzLB1ZdrDTwkvRSRFKuPywM92xAxdtKAPGbCoHn","type":"payment"}},"_links":{"next":{"href":"/transactions/{id}/operations?reverse=false\u0026cursor="},"prev":{"href":"/transactions/{id}/operations?reverse=true\u0026cursor="},"self":{"href":"/transactions/FSFsCjzLB1ZdrDTwkvRSRFKuPywM92xAxdtKAPGbCoHn/operations"}}}```
```